cypress/integration/api/auth.test.ts

```diff
@@ -0,0 +1,65 @@
+import { HttpMethod } from "@lib/http-method";
+import { getMagicLink } from "cypress/support/commands";
+import { TEST_USER } from "cypress/support/constants";
+import { StatusCodes } from "http-status-codes";
+import cookie from "cookie";
+
+describe("test send magic link api", () => {
+  beforeEach(() => {
+    cy.task("db:setup");
+  });
+
+  afterEach(() => {
+    cy.task("db:teardown");
+  });
+
+  it("should be able to send a magic link to a new user", () => {
+    cy.request({
+      method: HttpMethod.POST,
+      url: "/api/auth/magic",
+      body: {
+        macID: "macid",
+      },
+    }).then((response) => {
+      expect(response.status).equal(StatusCodes.OK);
+
+      cy.readMockData().then((data) => {
+        const magicLink = getMagicLink(data);
+        expect(magicLink).to.exist;
+      });
+    });
+  });
+
+  it("should be able to send a magic link to an existing user", () => {
+    cy.request({
+      method: HttpMethod.POST,
+      url: "/api/auth/magic",
+      body: {
+        macID: TEST_USER.id,
+      },
+    }).then((response) => {
+      expect(response.status).equal(StatusCodes.OK);
+
+      cy.readMockData().then((data) => {
+        const magicLink = getMagicLink(data);
+        expect(magicLink).to.exist;
+      });
+    });
+  });
+
+  it.only("should be able to logout", () => {
```

Comment:
Any reason for using only here? Do the other tests work?

Reply:
oh,forgot to take that out, thanks